src/components/Upload/UploadForm.js

```diff
@@ -114,37 +114,37 @@ class UploadForm extends Component {
 		let packageTypeOtherError = isFieldTouched('packageTypeOther') && getFieldError('packageTypeOther');
 		
 		return (
-			<div>
-				<Form >
+			<article className="container-fluid">
```

Comment:
Did we need 2 container-fluids here?

Reply:
It does look silly, but it compensates for a padding artifact the way the styles are currently written.  I wouldn't endorse this as a permanent fix.  It will be going away with KPMP-582 and 754.